shiftcommerce/flex-ruby-gem
# Issue/167 test mode always set to true

### The issue

Test mode was always being set to true,  this caused an error on live as basing on the result of the call to `test_mode` we decide which paypal endpoint to use.

### The fix

Revert the change so that the correct boolean will be returned.

### QA 

Using create a branch on matalan-rails-site which uses this branch:

`gem "flex_commerce_api", git: "https://github.com/shiftcommerce/flex-ruby-gem.git", branch: "issue/167_test_mode_always_set_to_true"`

deploy this branch to the live staging environment,

using the live staging url add an item to your basket and continue through checkout,

on the final step of checkout click the paypal button and you should be redirected to a paypal page,

DO NOT COMPLETE THE ORDER as this is a live url the order will actually be placed and you will be charged.

The front end branch release/1.75.5 contains this change and has been deployed to the staging url.
